visualisations/union-budget-board-2019/3-balance-column/index.html

```diff
@@ -0,0 +1,387 @@
+    }
+
+    .receipt-sub-vis-containner,
+    .exp-sub-vis-containner {
+        text-align: left;
+        margin-top: 30px;
+        font-weight: 700;
+        margin-left: 20px;
+        font-size: 16px;
+        color: rgba(0, 0, 0, 0.61);
+    }
+
+    .help-text {
+        text-align: left;
+        margin-left: 20px;
+        font-size: 11px;
+    }
+    </style>
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-lg-12 bg-container">
+            </div>
+        </div>
+        <div class="row">
+            <div class="container section-3col-box-wrapper">
+                <div class="row">
+                    <div class="col-sm-4 col-md-4 section-col1 ">
+                        <div class="white-box">
+                            <h2>Receipt Budget</h2>
+                            <div class="receipts-vis-container">
+                                <div class="receipt-summary-figs">
+                                    <div class="receipt-summary-num"></div>
+                                    <div class="receipt-summary-per-change"></div>
+                                </div>
+                                <div class="receipt-sub-vis-containner">
+                                    <div class="receipt-vis">
+                                        <div class="receipt-title-revenue"></div>
+                                        <div class="receipt-vis-revenue"></div>
+                                    </div>
+                                    <div class="receipt-vis">
+                                        <div class="receipt-title-capital"></div>
+                                        <div class="receipt-vis-capital"></div>
+                                    </div>
+                                </div>
+                                <div class="help-text">Note : Figures are in &#8377; Crores</div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-sm-4 col-md-4 section-col2">
+                        <div class="white-box">
+                            <h2>Union Budget 2019</h2>
+                            <hr class="hr-style-1" />
+                            <p class="obi-aim">At Open Budgets India it is our endeavour to make India's Budgets open, usable and easy to comprehend. Explore Expenditure Budget, Receipts Budget, Budget at a Glance and other key datasets related to Union Budget 2019-20 at:
+                                <a href="https://openbudgetsindia.org/" target="_blank"><img class="obi-logo" src="https://raw.githubusercontent.com/cbgaindia/design/master/logo_OBI/logo_types/light_bg_logo/draft_final/logo_with_text/draft_final.png" /></a> </p>
+                            <div class="btn-group" role="group" aria-label="...">
+                                <a href="https://openbudgetsindia.org/dataset?tags=union+budget+2019+interim" target="_blank">
```

Comment:
@shreyaagrawal0809  This URL is outdated referring to interim, please update it to: https://openbudgetsindia.org/dataset?tags=union+budget+2019+2020

Reply:
Done